src/Microsoft.Restier.Core/ApiConfiguratorAttribute.cs

```diff
@@ -115,6 +145,22 @@ protected ApiConfiguratorAttribute()
         }
 
         /// <summary>
+        /// Configures an API builder.
+        /// </summary>
+        /// <param name="builder">
+        /// An API builder.
+        /// </param>
+        /// <param name="type">
+        /// The API type on which this attribute was placed.
+        /// </param>
+        [CLSCompliant(false)]
+        public virtual void ConfigureBuilder(
```

Comment:
Sure. But I'm afraid merge with the Configure is not possible, since ApiConfiguration is built from ApiBuilder, they can't be configurable at the same time.